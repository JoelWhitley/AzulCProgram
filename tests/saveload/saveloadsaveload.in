1
Player 1
Player 2
turn 1 r 2
turn 2 r 3
turn 3 b 1
turn 4 l 1
save
test.save
exit
2
test.save
save
test2.save
exit
4
