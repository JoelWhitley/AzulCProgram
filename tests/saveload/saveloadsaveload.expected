Player 1
Player 2
0
0
Player 2
F Y B L U Y B B U R 
. . . . 
. . . . 
L U R Y 
. . . . 
Y B L U 
b y r u l 
l b y r u 
u l b y r 
r u l b y 
y r u l b 
b y r u l 
l b y r u 
u l b y r 
r u l b y 
y r u l b 
L 
R . 
. . . 
. . . . 
. . . . . 
. 
. . 
R . . 
. . . . 
. . . . . 
. . . . . . . 
. . . . . . . 

R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U R Y B L U 
0